### ISSUE TITLE
More of a suggestion/Question......

### ISSUE THREAD

**Issue by guydht** [pheyvaer/kitsu-tutorial#2]
When getting the authentication token for the user (which isn't written anywhere at the docs (anywhere that I looked at)) that you need for updating your list and stuff, Is it possible to also get that user's ID? As in, when I request "https://kitsu.io/api/oauth/token?grant_type=password&username=MYUSERNAME&password=MYPASSWORD"
it gives me the token but there's no way (that I know of....) of getting the ID for that same user (which is needed for updating the user's library). Of course you could just do another request to get the ID, but I think it should be obvious/implied that if someone wants a user's token he'd also want his ID.

**Comment by wopian** [comment]
Use the `self` filter: `api/edge/users?filter[self]=true` 👍

**Comment by guydht** [comment]
Yeah, I know that, but you need to do another request just for the ID...... What I'm saying is that the token request should also return the user's ID, or at least have an option to tell it to, to save the extra request, since most of the time if someone needs the token, he'll need the ID of that user.

**Comment by wopian** [comment]
That's currently not possible, as its not a standard property of oAuth2

Once proper oAuth2 with scopes and (actual) app-specific `clientId` is implemented soon, it could be an extra addition that gets injected into the returned response. E.g:

```
{
  "access_token":"036b9495-16d3-476e-b53a-ed5dfc7d6513",
  "expires_in":7199,
  "refresh_token":"811235f4-b39d-4626-b940-34399e7bc737",
  "token_type":"Bearer",
  "user_id":"2",
}
```

Although this is better suited at https://github.com/hummingbird-me/hummingbird

**Comment by guydht** [comment]
Okay, thanks for explaining..... I didn't post it on the main Kitsu page because I thought it's more of a back-end problem rather than a front-end, but I guess I'm wrong. Next time I'll write on [that](https://github.com/hummingbird-me/hummingbird) thread.

**Comment by Samanw** [comment]
Hi, is it possbile to get a anime myanimelist ID?
